changelog.md

```diff
@@ -7,6 +7,50 @@ All notable end-user facing changes are documented in this file.
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
 
+### [3.2.0-rc2] (changes since 3.2.0-rc1)
+
+- Non-breaking bug fixes
+  - Do not reject rejection messages locally. Drop them instead (#6525)
+
+### [3.1.7] (changes since 3.1.6)
```

Comment:
this one is out of order... shouldn't it be above 3.1.6?

Reply:
I'm not sure I understand - it is above 3.1.6. That's the order we had them released: 3.1.6 -> 3.2.0-rc1 -> 3.1.7 -> 3.2.0-rc2.

Reply:
hmm, ok, I didn't realize you order them by date. As a user I would expect the changelog to be ordered by version, regardless of date, as by the time I would be considering whether to update to certain version, I don't care about dates, but about evolution of the different major (or minor since this is not following semVer) versions, which is where there are breaking changes.